### DISCUSSION TITLE
solve BTE on top of scph solution

### DISCUSSION THREAD

**Discussion by zhibingao** [ttadano/alamode#21] [Q&A] [answered]
Dear Tadano,

After renormalization using dfc2, one can obtain the new second-order FCs. But I do not quite understand the logic.

Next, we can compute the kappa. What about the input? Here, there are two choices. One is the renormalized 2FCs + conventional (old) 3FCs. The other one is renormalized 2FCs + high order FCs using SCPH. In the latter one, it means new 3FCs, 4FCs and even 6FCs are included. Which one is true? Besides, dfc2 needs a specific temperature. It should be varied since different kappa needs different dfc2. Please be free to give comments and suggestions. 

Best wishes,
Zhibin

**Comment by ttadano** [comment, marked as answer]
Hi,

In the current implementation of alamode, only the former choice, 

- renormalized 2FCs + conventional (old) 3FCs, 

is supported. Another possible choice may be 

- renormalized 2FCs + renormalized 3FCs,

but it is not supported within alamode yet.

When ``MODE = RTA``, the anphon code only reads the 2FCs and 3FCs from given XML files. By contrast, when ``MODE = SCPH``, the code additionally reads the 4FCs because they are necessary to compute the renormalized 2FCs based on the self-consistent phonon theory.
Therefore, the second combination of

- renormalized 2FCs + high order FCs using SCPH

is not possible.

> Besides, dfc2 needs a specific temperature. It should be varied since different kappa needs different dfc2.

Yes. There are two temperatures, one is the temperature in the thermal conductivity calculation and the other is the temperature of the SCPH calculation. The former, T_kappa, affects the Bose-Einstein distribution function of each phonon mode with a fixed frequency in the thermal conductivity calculation, whereas the latter, T_scph, controls the amplitude of the thermal fluctuation in the self-consistent phonon calculations.  

In the ``dfc2`` program, you need to give T_scph. 

When calculating the thermal conductivity based on
- renormalized 2FCs + conventional (old) 3FCs, 

you will have to set T_scph = T_kappa.
